swagger.yaml

```diff
@@ -0,0 +1,515 @@
+swagger: "2.0"
+info:
+  version: "v3"
+  title: "Billing Account API"
+# during production,should point to your server machine
+host: localhost:3000
+basePath: "/v3"
+# during production, should use https
+schemes:
+- "http"
+produces:
+- application/json
+consumes:
+- application/json
+
+securityDefinitions:
+  Bearer:
+    type: apiKey
+    name: Authorization
+    in: header
+
+paths:
+  /billingaccounts:
+    get:
+      tags:
+        - billingaccount
+      security:
+        - Bearer: []
+      operationId: findBillingAccounts
+      description: Retreive billing accounts that match the filter, Admin user only.
+      responses:
+        '403':
+          description: No permission or wrong token
+          schema:
+            $ref: "#/definitions/ErrorModel"
+        '422':
+          description: Invalid input
+          schema:
+            $ref: "#/definitions/ErrorModel"
+        '200':
+          description: A list of billing accounts
+          schema:
+            $ref: "#/definitions/BillingAccountListResponse"
+      parameters:
```

Comment:
@skyhit Can you add this?  

Given a billing account, return the Direct projects that are associated to it.